### PR TITLE
use torch.long for multiclass classification labels

### PR DESCRIPTION
**Pull request by raphaelschwinger**

### CODE REVIEW

File: birdset/datamodule/components/transforms.py [hunk 1]
```diff
@@ -276,7 +276,10 @@ def transform_values(self, batch):
         input_values = waveform_batch["input_values"]
         input_values = input_values.unsqueeze(1)
         labels = torch.tensor(batch["labels"])
+<<<<<<< HEAD
```
Comment (raphaelschwinger) [inline birdset/datamodule/components/transforms.py:279]:
```suggestion
```

File: birdset/datamodule/components/transforms.py [hunk 2]
```diff
@@ -276,7 +276,10 @@ def transform_values(self, batch):
...
+=======
 
+>>>>>>> origin/main
```
Comment (raphaelschwinger) [inline birdset/datamodule/components/transforms.py:282]:
```suggestion
```

File: birdset/datamodule/components/transforms.py [hunk 3]
```diff
@@ -481,6 +484,76 @@ def _prepare_call(self):
...
+
+        waveform_batch = self.feature_extractor(
+            waveform_batch,
+            padding="max_length",
+            max_length=self.max_length,
+            truncation=True,
+            return_attention_mask=True,
+        )
+        return waveform_batch
+=======
```
Comment (raphaelschwinger) [inline birdset/datamodule/components/transforms.py:556]:
```suggestion
```
